java/Combination_Sum.md

```diff
@@ -0,0 +1,104 @@
+| id               | sidebar_position | title            | sidebar_label      | description                                                                                                                                              | tags                          |
+|------------------|------------------|------------------|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
+| combination-sum  | 2                | Combination Sum   | Combination Sum      | Given an array of distinct integers candidates and a target integer target, return a list of all unique combinations of candidates where the chosen numbers sum to target. | java, problem-solving         |
+
```

Comment:
Please follow our documentation format and syntax. Also, please read our contribution guidelines carefully.